src/main/java/com/deque/html/axecore/extensions/WebDriverInjectorExtensions.java

```diff
@@ -188,6 +191,8 @@ private static void injectIntoFramesAsync(final WebDriver driver,
         driver.switchTo().frame(frame);
 
         js.executeAsyncScript(script);
+        js.executeScript("axe.configure({ allowedOrigins: ['<unsafe_all_origins>'] });");

```

Comment:
Why is this being done here? This will cause the line to be run every time a piece of code is injected, not just after axe.